.github/workflows/doc-problem-match.json

```diff
@@ -0,0 +1,16 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "latex-parser",
+      "pattern": [
+        {
+          "regexp": "^\\[LATEX(WARNING|ERROR)::(.*?):(\\d+)::(.*?)\\]$",
+          "severity": 1,
+          "file": 2,
+          "line": 3,
+          "message": 4
+        }
+      ]
+    }
+  ]
+}
```

Comment:
This is pretty cool.  It would be nice to adapt the pattern matching that we use in Decent CI into our other builds.